demo.sh

```diff
@@ -4,29 +4,37 @@ set -e
 
 # include the goodness
 source ./init.sh
-REPO=https://github.com/18F/cf-sample-app-spring
-DIR=cf-sample-app-spring
-APP=cf-spring
+REPO=https://github.com/pburkholder/sample-app
+DIR=sample-app
+APP=sample-app
 
 cleanup() {
   echo "DOING PRE-DEMO CLEAN UP" 
-  cf delete cf-spring -f >/dev/null
-  cf delete-service cf-spring-db -f >/dev/null
+  cf delete sample-app -f >/dev/null
+  # cf delete-service sample-app-db -f >/dev/null
```

Comment:
I assume we do want to delete the sample database?

Reply:
The clean up is run before every demo.sh, and the DB creation now takes ten minutes, so we only want to delete the DB when running `demo-cleanup.sh`.